lib/fluent/plugin/out_lm.rb

```diff
@@ -77,7 +79,7 @@ def start
       @http_client = Net::HTTP::Persistent.new name: "fluent-plugin-lm-logs", proxy: proxy_uri
       @http_client.override_headers["Content-Type"] = "application/json"
       @http_client.override_headers["User-Agent"] = log_source + "/" + LmLogsFluentPlugin::VERSION
-      @url = "https://#{@company_name}.logicmonitor.com/rest/log/ingest"
```

Comment:
I think `logicmonitor.com` should come from the variable.

Reply:
Yes line 82 is the change
@url = "https://#{@company_name}.#{@company_domain}.com/rest/log/ingest"